atomix/pom.xml

```diff
@@ -0,0 +1,283 @@
+
+    <prometheus.client.version>0.8.1</prometheus.client.version>
+    <slf4j.version>1.7.30</slf4j.version> <!-- Overridden in some profiles -->
+    <version.errorprone>2.3.4</version.errorprone>
+    <version.objenesis>3.1</version.objenesis>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <artifactId>jackson-bom</artifactId>
+        <groupId>com.fasterxml.jackson</groupId>
+        <scope>import</scope>
+        <type>pom</type>
+        <version>${jackson.version}</version>
+      </dependency>
+
+
+      <!-- ATOMIX DEPENDENCIES -->
+      <dependency>
+        <artifactId>atomix-cluster</artifactId>
+        <groupId>io.zeebe</groupId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>atomix-storage</artifactId>
+        <groupId>io.zeebe</groupId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>atomix-utils</artifactId>
+        <groupId>io.zeebe</groupId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>atomix</artifactId>
+        <groupId>io.zeebe</groupId>
+        <version>${project.version}</version>
+      </dependency>
+
+      <dependency>
+        <artifactId>slf4j-api</artifactId>
+        <groupId>org.slf4j</groupId>
+        <version>${slf4j.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>junit</artifactId>
+        <groupId>junit</groupId>
+        <scope>test</scope>
+        <version>${junit.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>guava-testlib</artifactId>
+        <groupId>com.google.guava</groupId>
+        <scope>test</scope>
+        <version>${version.guava}</version>
+      </dependency>
+      <dependency>
+        <artifactId>concurrentunit</artifactId>
+        <groupId>net.jodah</groupId>
+        <scope>test</scope>
+        <version>${concurrentunit.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>mockito-core</artifactId>
+        <groupId>org.mockito</groupId>
+        <scope>test</scope>
+        <version>${version.mockito}</version>
+      </dependency>
+      <dependency>
+        <artifactId>guava</artifactId>
+        <groupId>com.google.guava</groupId>
+        <version>${version.guava}</version>
+      </dependency>
+      <dependency>
+        <artifactId>commons-lang3</artifactId>
+        <groupId>org.apache.commons</groupId>
+        <version>${commons.lang3.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>commons-math3</artifactId>
+        <groupId>org.apache.commons</groupId>
+        <version>${commons.math3.version}</version>
+      </dependency>
+      <dependency>
+        <artifactId>kryo</artifactId>
+        <groupId>com.esotericsoftware</groupId>
+        <version>${kryo.version}</version>
```

Comment:
I don't see us getting rid of Kryo for a while - should we add it to our own parent pom?